src/components/NavBar.tsx

```diff
@@ -6,17 +6,23 @@ import React from "react";
 
 export function NavBar() {
     const [closeBurger, setBurgerState] = useState(true);
+    const [currentPage, setCurrentPage] = useState("Home");
+
+    function clickLinkHandler (currentPage: string) {
+        setBurgerState(true); 
+        setCurrentPage(currentPage);
+    }
 
     return (
         <div className='header'>
             <div className={closeBurger ? "nav-bar" : "nav-bar-mobile"} data-testid="nav-bar">
-                <Link to='/' className = 'nav-link'> Home </Link>
-                <Link to='/spirit' className = 'nav-link'>Spirit </Link>
-                <Link to='/opportunity'className = 'nav-link'>Opportunity</Link>
-                <Link to='/curiosity'className = 'nav-link'>Curiosity</Link>
-                <Link to='/timeline'className = 'nav-link' > Timeline</Link>
+                <Link to='/' className = 'nav-link' data-testid="home" onClick={()=>{clickLinkHandler("Home");}}> Home </Link>
```

Comment:
```suggestion
                <Link to='/' className = 'nav-link' data-testid="home" onClick={()=>clickLinkHandler("Home")}> Home </Link>
```
this would tidy slightly :)

Reply:
Without semicolons it does not compile (probably lint)

Reply:
Failed to compile
src\components\NavBar.tsx
  Line 19:110:  Missing semicolon  semi